testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeAlterTableCompatibility.java

```diff
@@ -167,4 +172,33 @@ public void testCommentOnColumnUnsupportedWriterVersion()
             onTrino().executeQuery("DROP TABLE delta.default." + tableName);
         }
     }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS})
```

Comment:
Is there no need to add `PROFILE_SPECIFIC_TESTS`?

Reply:
Thanks, there is very much a need